tests/__init__.py

```diff
@@ -24,3 +24,13 @@ def test_freeze_tick(self):
             dt = datetime.datetime.utcnow()
             self.assertTrue(datetime.datetime(2014, 1, 1) < dt < datetime.datetime(2014, 1, 1, 0, 0, 1))
             self.assertTrue(1388534400 < time.time() < 1388534401)
+
+    def test_now(self):
+        """
+        utcnow() should always be preferred over now() and hence now() should
```

Comment:
not sure if `utcnow()` is actually always _preferred_ over `now()` since you can do `now(tz=pytz.UTC)` to get a timezone aware dt